redaxo/src/addons/be_style/README.md

```diff
@@ -0,0 +1,50 @@
+be_style-Addon
+=============
+
+Dieses Addon stellt die Styles, Skripte und Grafiken für das REDAXO Backend bereit.
+
+## Styles kompilieren
+
+Bei Änderungen müssen die Styles neu kompiliert werden. Hierfür stellt **be_style** mehrere Varianten zur Verfügung.
+
+### 1. via REDAXO Konsole
+Über die Konsole von REDAXO können die Stles neu kompiliert werden. Hierfür benutzt man den von **be_style** bereitgestellten Kompilierungsvorgang via `be_style:compile` oder `styles:compile`.
+
+### 2. via package.yml
+In der `package.yml` von **be_style** und von den PlugIn **redaxo** kann der Wert von `compile` von `0` auf `1` gesetzt werden.
```

Comment:
Soweit ich thomas verstanden habe muss man in der regel nur im redaxo plugin das compile flag einschalten